terminaltables.py

```diff
@@ -22,21 +22,56 @@
 import re
 import struct
 import sys
+import unicodedata
+
+PY2 = sys.version_info[0] == 2
 
 try:
     import fcntl
     import termios
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
This is already covered by Windows tests on appveyor. Coverage gets merged in codecov: https://codecov.io/github/Robpol86/terminaltables/terminaltables.py?ref=master no need for this comment.